packages/lab/src/__tests__/stepped-tracker/SteppedTracker.cy.tsx

```diff
@@ -0,0 +1,208 @@
+import { Step, SteppedTracker } from "@salt-ds/lab";
+import { composeStories } from "@storybook/react";
+
+import * as steppedTrackerStories from "@stories/stepped-tracker/stepped-tracker.stories";
+
+const { VerticalDepth1, VerticalDepth2 } = composeStories(
+  steppedTrackerStories,
+);
+
+describe("<SteppedTracker />", () => {
+  it("should expand/collapse when trigger is clicked (depth 1)", () => {
+    cy.mount(<VerticalDepth1 />);
+
+    cy.get("#step-1").should("be.visible");
```

Comment:
is there no other way to get to these? avoid using id at all cost

Reply:
removed `.get()` from the first 2 tests, but kept them in the a11y ones

Reply:
removed a11y ones